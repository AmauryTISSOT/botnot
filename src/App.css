:root {
  --font-color: #333;
  --background-color: #eee;
  --link-color: cornflowerblue;
  --menu-color : cornflowerblue;
}

[data-theme="dark"] {
  --font-color: #eee;
  --background-color: #333;
  --link-color: lightblue;
  --menu-color: rgb(75, 65, 65) ;
}

a {
  color: var(--link-color);
}

.App {
  text-align: center;
  background-color: var(--background-color);
  background-size: cover;
  height: 100vh;
  color: var(--font-color);
}

html {
   min-height: 100%;
}