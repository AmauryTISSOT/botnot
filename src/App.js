import "./App.css";
import { Routes, Route } from "react-router-dom";
import Home from "./components/Home/Home";
import About from "./components/About/About";
import NoMatch from "./components/NoMatch/NoMatch";
import Annuaire from "./components/Annuaire/Annuaire";
import CustomerQuiz from "./components/CustomerQuiz";
import Outils from "./components/Outils/Outils";

function App() {
  return (
    <div className="App">
      <Routes>
        <Route path="/" element={<Home />} />
        <Route path="about" element={<About />} />
        <Route path="*" element={<NoMatch />} />
        <Route path="outils" element={<Outils />}>
          <Route path="annuaire" element={<Annuaire />} />
          <Route path="customerquiz" element={<CustomerQuiz />} />
        </Route>
      </Routes>
    </div>
  );
}

export default App;
