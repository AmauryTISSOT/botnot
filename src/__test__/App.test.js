import { render, screen } from '@testing-library/react';
import App from '../App';

describe("App component", () => {
  test("App render correctly", () => {
    const {container} = render(<App/>);
    expect(container).toMatchSnapshot();
  })
})