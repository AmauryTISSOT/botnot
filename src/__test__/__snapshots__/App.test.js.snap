// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component App render correctly 1`] = `
<div>
  <div
    class="App"
  >
    <div>
      <div>
        <div>
          <div>
            <div>
              <div>
                <h3>
                  Avez-vous connaissance de servitudes particulières sur votre propriété ? (servitude de passage, d'alignement, etc.)
                </h3>
                <p>
                  id keystone : 
                  servitude
                </p>
                <div>
                  <label>
                    <input
                      id="servitude"
                      type="checkbox"
                      value="true"
                    />
                    oui
                  </label>
                  <label>
                    <input
                      id="servitude"
                      type="checkbox"
                      value="false"
                    />
                    non
                  </label>
                </div>
              </div>
            </div>
            <div>
              <div>
                <h3>
                  Avez-vous un mur séparatif avec vos voisins ?
                </h3>
                <p>
                  id keystone : 
                  murSeparatif
                </p>
                <div>
                  <label>
                    <input
                      id="murSeparatif"
                      type="checkbox"
                      value="true"
                    />
                    oui
                  </label>
                  <label>
                    <input
                      id="murSeparatif"
                      type="checkbox"
                      value="false"
                    />
                    non
                  </label>
                </div>
              </div>
            </div>
            <div>
              <div>
                <h3>
                  Votre terrain est-il en pente ?
                </h3>
                <p>
                  id keystone : 
                  terrainEnPente
                </p>
                <div>
                  <label>
                    <input
                      id="terrainEnPente"
                      type="checkbox"
                      value="true"
                    />
                    oui
                  </label>
                  <label>
                    <input
                      id="terrainEnPente"
                      type="checkbox"
                      value="false"
                    />
                    non
                  </label>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
